### PR TITLE
CNV GetHetCoverage: mention tool is non-bayesian version of GetBeyesianHetCoverage

### PR DESCRIPTION
**Pull request by sooheelee**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java [hunk 1]
```diff
@@ -20,8 +20,10 @@
 import java.io.File;
 
 /**
- * Outputs reference/alternate read counts at heterozygous SNP sites present in a normal sample
- * (and at the same sites in a tumor sample, if specified).
+ * Outputs reference/alternate read counts at heterozygous SNP sites present in a normal sample,
+ * and, if specified, at the same sites in a tumor sample.
+ *
+ * <p>The non-Bayesian version of {@link GetBayesianHetCoverage}. See GetBayesianHetCoverage for modes.</p>
```
Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java]: What is meant by "modes"?

Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java]: The tumor_only, normal_only and matched normal and tumor modes

File: src/main/java/org/broadinstitute/hellbender/tools/copynumber/CollectAllelicCounts.java [hunk 2]
```diff
@@ -113,4 +113,4 @@ private void validateArguments() {
         ParamUtils.isPositiveOrZero(minimumMappingQuality, "Mapping-quality threshold must be greater than or equal to zero.");
         ParamUtils.isPositiveOrZero(minimumBaseQuality, "Base-quality threshold must be greater than or equal to zero.");
     }
-}
\ No newline at end of file
+}
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/copynumber/CollectAllelicCounts.java]: Seems I can't remove the new line at the end here.

File: src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java]: Let's continue this discussion after the two repo merges.

**Pull request merged**